packages/valory/skills/safe_deployment_abci/rounds.py

```diff
@@ -71,47 +72,47 @@ class RandomnessSafeRound(CollectSameUntilThresholdRound):
     """A round for generating randomness"""
 
     allowed_tx_type = RandomnessPayload.transaction_type
-    payload_attribute = "randomness"
+    payload_attribute = get_name(RandomnessPayload.randomness)
```

Comment:
Here benefit should be obvious. Impossible to pass static checks with misdefined properties.